pos_frequency_pipeline.py

```diff
@@ -2,7 +2,60 @@
 Implementation of POSFrequencyPipeline for score ten only.
 """
 
+import json
+import re
+
+from pymystem3 import Mystem
+
+from constants import ASSETS_PATH
+from pipeline import CorpusManager
+from visualizer import visualize
+
 
 class POSFrequencyPipeline:
     def __init__(self, assets):
-        pass
+        self.assets = assets
+        self.pos_frequencies = {}
+        self._text = ''
+
+    def run(self):
+        storage = self.assets.get_articles()
+
+        for article_id, article in storage.items():
+            self._text = article.get_raw_text()
+            self._process()
+            self._add_pos_to_metadata(article_id)
+            visualize(statistics=self.pos_frequencies, path_to_save=f'{ASSETS_PATH}\\{article_id}_image.png')
+
+    def _process(self):
+        result = Mystem().analyze(self._text)
```

Comment:
the pipeline does not analyze the text - it works on top of already processed files. change the logic accordingly